amr-wind/wind_energy/actuator/disk/disk_ops.cpp

```diff
@@ -126,7 +127,7 @@ void collect_parse_conflicts(
     std::ostringstream& ss)
 {
     if (pp.contains(p1) && pp.contains(p2)) {
-        ss << "UniformCt Conflict: " << p1 << " and " << p2 << std::endl;
+        ss << "ActuatorDisk Conflict: " << p1 << " and " << p2 << std::endl;
```

Comment:
I'm wondering if it might be best to use an identifier in here instead of spelling "ActuatorDisk" all the time?

Reply:
Sure. We can add an argument for the identifier. This is a free function so it has no idea what the identifier is for the object calling it which is why I generalized it, but it would make things cleaner to give a specific identifier as you suggest.